plugins/headlamp-plugin/plugin-management/plugin-management.js

```diff
@@ -237,6 +242,31 @@ class PluginManager {
   }
 }
 
+/**
+ * Checks the plugin name is a valid one.
+ *
+ * Look for "..", "/", or "\" in the plugin name.
+ *
+ * @param {string} pluginName
+ *
+ * @returns true if the name is valid.
+ */
+function validatePluginName(pluginName) {
+  const invalidPattern = /[\/\\]|(\.\.)/;
+  return !invalidPattern.test(pluginName);
+}
+
+/**
+ * @param {string} archiveURL - the one to validate
+ * @returns true if the archiveURL looks good.
+ */
+function validateArchiveURL(archiveURL) {
+  return (
+    archiveURL.startsWith('https://artifacthub.io/packages/') ||
+    archiveURL.startsWith('https://github.com/yolossn/headlamp-plugins/')
```

Comment:
Is this URL required?

Reply:
I could see the tests were using this URL currently. Probably we should move the test plugins to the plugins repo or the headlamp repo in the future.